bindings/python/CMakeLists.txt

```diff
@@ -130,19 +130,18 @@ IF(BUILD_PYTHON_INTERFACE)
 
   SET(PKG_CONFIG_PYWRAP_REQUIRES "eigenpy >= 2.6.2")
   ADD_LIBRARY(${PYWRAP} SHARED ${${PYWRAP}_SOURCES} ${${PYWRAP}_HEADERS})
+  MODERNIZE_TARGET_LINK_LIBRARIES(${PYWRAP} SCOPE PUBLIC
+    TARGETS eigenpy::eigenpy)
   IF(BUILD_WITH_OPENMP_SUPPORT)
     TARGET_COMPILE_OPTIONS(${PYWRAP} PRIVATE ${OpenMP_CXX_FLAGS})
     TARGET_COMPILE_DEFINITIONS(${PYWRAP} PRIVATE -DPINOCCHIO_PYTHON_INTERFACE_WITH_OPENMP)
     TARGET_INCLUDE_DIRECTORIES(${PYWRAP} SYSTEM PRIVATE ${OpenMP_CXX_INCLUDE_DIR})
     IF(LINK_PYTHON_INTERFACE_TO_OPENMP)
       TARGET_LINK_LIBRARIES(${PYWRAP} PRIVATE ${OpenMP_CXX_LIBRARIES})
     ENDIF(LINK_PYTHON_INTERFACE_TO_OPENMP)
-    MESSAGE(STATUS "OpenMP_CXX_LIBRARIES: ${OpenMP_CXX_LIBRARIES}")
```

Comment:
It seems this got removed by mistake, right?

Reply:
It was for my own debug purpose. This is not needed.